check.py

```diff
@@ -14,30 +14,35 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import os, shutil, sys, subprocess, difflib, json, time, urllib2
+import os, shutil, sys, subprocess, difflib, json, time, urllib2, optparse
```

Comment:
`optparse` is not obsolete, it is just no longer developed.. but sure, I'll convert.